src/core/lib/iomgr/tcp_posix.cc

```diff
@@ -381,10 +385,15 @@ class TcpZerocopySendCtx {
   //
   // Please refer to the STATE TRANSITION DIAGRAM below for more details.
   //
-  bool UpdateZeroCopyOMemStateAfterSend(bool seen_enobuf) {
+  bool UpdateZeroCopyOMemStateAfterSend(bool seen_enobuf, bool& cont) {
```

Comment:
See comment below. Can we just elide this parameter by returning "true" in any scenario where we would have set this to true?